pmd-cs/src/main/antlr4/net/sourceforge/pmd/lang/cs/antlr4/CSharpLexer.g4

```diff
@@ -19,7 +19,8 @@ private boolean verbatium;
 BYTE_ORDER_MARK: '\u00EF\u00BB\u00BF';
 
 SINGLE_LINE_DOC_COMMENT: '///' InputCharacter*    -> channel(COMMENTS_CHANNEL);
-DELIMITED_DOC_COMMENT:   '/**' .*? '*/'           -> channel(COMMENTS_CHANNEL);
+EMPTY_DELIMITED_DOC_COMMENT: '/***/'              -> channel(COMMENTS_CHANNEL);
```

Comment:
@maikelsteneker Thanks for adding tests! The one last thing I have reservation is this change, as it's not obvious to me what is fixed here. Was `/***/` incorrectly tokenized previously? Maybe, add this to the test case and this should be gold.

Reply:
I've added a test to cover this case. It fails with the old tokenizer, but passes with this updated one.